src/ui.rs

```diff
@@ -209,7 +209,7 @@ impl ThemedWidget for &results::Results {
             .split(res_chunks[0]);
 
         let exit = Span::styled(
-            "Press 'q' to quit or 'r' for another test.",
+            "Press 'q' to quit or 'r' for another test or 'p' to practice missed words",
```

Comment:
Small grammar error. Should be "Press 'q' to quit, 'r' for another test, or 'p' to practice missed words."

Reply:
that is a nice touch.

to be honest I don't think even monkeytype has this feature.